bgpd/bgp_vty.c

```diff
@@ -19202,6 +19455,8 @@ int bgp_config_write(struct vty *vty)
 	if (bm->outq_limit != BM_DEFAULT_Q_LIMIT)
 		vty_out(vty, "bgp output-queue-limit %u\n", bm->outq_limit);
 
+	vty_out(vty, "!\n");
```

Comment:
This should be a separate commit at all, IMO.

Reply:
Do you mean a separate commit for all the changes in bgp_config_write() function? or just this particular line?

Reply:
This line only...

Reply:
Done